setup.cfg

```diff
@@ -43,4 +43,4 @@ install_requires = astropy
 # version should be PEP440 compatible (https://www.python.org/dev/peps/pep-0440/)
 version = 0.0.dev
 # Note: you will also need to change this in your package's __init__.py
-minimum_python_version = 3.7
+minimum_python_version = 3.6
```

Comment:
Do we need a max python version for the Pi? Or just remove these again if we don't really need astropy